contribute/design/cstor-operator/day2ops.md

```diff
@@ -0,0 +1,277 @@
+### Introduction
+
+This document focuses on cStor day 2 operations work flow via the proposed specs
+of CSPC and CSP mentioned in contribute/design/cstor-operator/specs.md
+
+### Pool Expansion
+
+* “As a OpenEBS user, I should be able to add block devices to CSPC to increase
+   the pool capacity.” *
+
+Steps To Be Performed By User:
+
+1. `kubectl edit cspc your_cspc_name`
+
+2. Put the block devices in the CSPC spec against the correct kubernetes nodes.
+
+
+**Consider following example for stripe pool expansion**
+
+Current CSPC is following:
+
+
+This CSPC corresponds to a stripe pool on node `gke-cstor-it-default-pool-1`
+
+```
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-stripe
+spec:
+  pools:
+  - nodeSelector:
+      kubernetes.io/hostName: gke-cstor-it-default-pool-1
+    raidConfig:
+    - type: stripe
+      isWriteCache:false
+      isSpare: false
+      isReadCache: false
+      blockDevices:
+      - blockDeviceName: sparse-3c1fc7491f9e4cf50053730740647318
+        id: bd1
+        capacity: 12345234
+    poolConfig:
+      cacheFile: var/openebs/disk-img-0
+      raidType: mirror
+      overProvisioning: false
+      compression: false
+
+```
+
+Expanding stripe pools -- the spec will look
+following:
+
+
+```
+
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-stripe
+spec:
+  pools:
+  - nodeSelector:
+      kubernetes.io/hostName: gke-cstor-it-default-pool-1
+    raidConfig:
+    - type: stripe
+      isWriteCache:false
+      isSpare: false
+      isReadCache: false
+      blockDevices:
+      - blockDeviceName: sparse-3c1fc7491f9e4cf50053730740647318
+        id: bd1
+        capacity: 12345234
+      // New block device added
+      blockDevices:
```

Comment:
I guess `blockDevices:` entry is not required

Reply:
:+1: not required